apps/authentication-portal/src/main/webapp/oauth2_consent.jsp

```diff
@@ -110,11 +111,14 @@
                                 <div class="scopes-list ui list">
                                     <%
                                         for (String scopeID : openIdScopes) {
+                                            String displayName = new OAuth2ScopeService()
+                                                                    .getScope(scopeID)
```

Comment:
Can't we have everything in one line?

Reply:
Then the line will exceed the 120 character limit.
![image](https://user-images.githubusercontent.com/20745428/73238246-48d91780-41be-11ea-8226-21715872b022.png)

Reply:
@thivi - no need to worry about character limit in jsp files as it contains html. Because sometime when we break html tags in to multiple, it's difficult to read the code. But since this is jsp part its not much an issue.